dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -13,14 +13,13 @@ import { enhanceTreats } from './enhanceTreats';
 import { groupCards } from './groupCards';
 
 const FORBIDDEN_CONTAINERS = [
-	'Palette styles new do not delete',
 	'Palette styles',
 	'culture-treat',
 	'newsletter treat',
 	'qatar treat',
 ];
 const isSupported = (collection: FECollectionType): boolean =>
-	!FORBIDDEN_CONTAINERS.includes(collection.displayName);
```

Comment:
I think all these palette containers all use the same Thrasher, so it might be a bit less brittle to check if any of the cards in the container use the thrashers embedUri.

Something like

```suggestion
!FORBIDDEN_CONTAINERS.includes(collection.displayName) && !collection.cards.find(card => card.embedUri === 'https://content.guardianapis.com/atom/interactive/interactives/2022/03/29/fronts-container-colours/default');
```

Reply:
Ah good to know! Yes that definitely feels like a better way of doing it and should avoid the issue with typos cropping up again